<?xml version="1.0" encoding="UTF-8"?><testrun name="TestRunner (1)" project="APIFramework" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="cucumber.Options.TestRunner" time="3.302">
    <testsuite name="Validating Place Api's" time="3.302">
      <testcase name="Verfiy if delete place functionality is working" classname="Validating Place Api's" time="3.302">
        <failure>java.lang.AssertionError: expected:&lt;404&gt; but was:&lt;200&gt;&#13;
	at org.junit.Assert.fail(Assert.java:88)&#13;
	at org.junit.Assert.failNotEquals(Assert.java:743)&#13;
	at org.junit.Assert.assertEquals(Assert.java:118)&#13;
	at org.junit.Assert.assertEquals(Assert.java:555)&#13;
	at org.junit.Assert.assertEquals(Assert.java:542)&#13;
	at stepDefinations.stepDefination.the_api_call_is_success_and_response_status_code(stepDefination.java:72)&#13;
	at âœ½.the API call is success and response status code 200(file:///D:/API%20Automatiom/APIFramework/src/test/java/features/placeValidation.feature:22)&#13;
&#13;
</failure>
      </testcase>
    </testsuite>
  </testsuite>
</testrun>
